src/utils/transformBundle.js

```diff
@@ -0,0 +1,28 @@
+export default function transformBundle ( source, transformers ) {
+	if ( typeof source === 'string' ) {
+		source = {
+			code: source,
+			map: null
+		};
+	}
+
+	return transformers.reduce( ( previous, transformer ) => {
+		let result = transformer( previous );
```

Comment:
I think composing sourcemaps should to transformer. Like with uglify we just pass previous map.